edward/criticisms/ppc_plots.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import seaborn as sns
```

Comment:
This adds an install requirement to Edward. I prefer minimal requirements, so can you wrap this in a try/except as follows?
```python
try:
  import seaborn as sns
except ImportError:
  pass
```